scripts/check.tokens.erc20.ts

```diff
@@ -0,0 +1,43 @@
+import type { Erc20Contract } from '$eth/types/erc20';
+import {
+	ERC20_CONTRACTS_PRODUCTION,
+	ERC20_CONTRACTS_SEPOLIA,
+	ERC20_TWIN_TOKENS_MAINNET,
+	ERC20_TWIN_TOKENS_SEPOLIA
+} from '../src/frontend/src/env/tokens.erc20.env';
+
+const getAddresses = (contracts: Erc20Contract[]) =>
+	contracts.map((contract) => contract.address.toLowerCase());
+
+const productionAddresses = getAddresses(ERC20_CONTRACTS_PRODUCTION);
+const twinTokensMainnetAddresses = getAddresses(ERC20_TWIN_TOKENS_MAINNET);
+
+const sepoliaAddresses = getAddresses(ERC20_CONTRACTS_SEPOLIA);
+const twinTokensSepoliaAddresses = getAddresses(ERC20_TWIN_TOKENS_SEPOLIA);
+
+const checkDuplicates = (addresses1: string[], addresses2: string[]) =>
```

Comment:
NTH: This function does not really "check", it "filter" the addresses

Reply:
right! naming!